lib/file_previewer.dart

```diff
@@ -64,14 +63,12 @@ class FilePreview {
       );
       return image != null
           ? Image.memory(image.bytes)
-          : defaultImage ?? _defaultImage(filePath);
+          : defaultImage ?? _defaultImage;
     } catch (e) {
-      return defaultImage ?? _defaultImage(filePath);
+      return defaultImage ?? _defaultImage;
     }
   }
 
   /// In case a file preview cannot be properly rendered, show a placeholder image with the extension in the center
```

Comment:
And no default image is provided

Reply:
so developers can't pass them in case they want to use custom one?

Reply:
ah, sorry, misunderstood:sweat_smile:

Reply:
No problem!